tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py

```diff
@@ -388,24 +353,32 @@ def cat_value_fn(idx: int) -> float:
     assert categorical_suggestion in categories
 
 
-def test_multi_objective_sample_int_uniform_distributions() -> None:
+@pytest.mark.parametrize(
+    "log, step",
+    [
+        (False, 1),
+        (True, 1),
+        (False, 2),
+    ],
+)
+def test_multi_objective_sample_int_distributions(log: bool, step: int) -> None:
     """Test sampling from int distribution returns integer."""
 
     study = optuna.create_study(directions=["minimize", "maximize"])
     random.seed(128)
 
     def int_value_fn(idx: int) -> float:
-        return random.randint(0, 100)
+        return random.randint(1, 100)
 
-    int_dist = optuna.distributions.IntDistribution(1, 100)
+    int_dist = optuna.distributions.IntDistribution(1, 100, log, step)
     past_trials = [
         frozen_trial_factory(
             i, [random.random(), random.random()], dist=int_dist, value_fn=int_value_fn
         )
         for i in range(16)
     ]
 
-    trial = frozen_trial_factory(16, [0, 0])
+    trial = frozen_trial_factory(16, [1, 1])
```

Comment:
```suggestion
    trial = frozen_trial_factory(16, [0, 0])
```

Please correct me if I missed something but I think we don't have to change here (it's values of an objective but parameters).

Reply:
Yes, let me revert the change. Thank you.